addons/project/static/src/components/project_task_state_selection/project_task_state_selection.scss

```diff
@@ -7,7 +7,7 @@
     }
 
     .fa-lg {
-        font-size: 1.75em;
+        font-size: 1.57em;
```

Comment:
Why did we change that? 🧐

Reply:
The state icon in the pop is getting clipped something like this
![2024-07-26_16-01](https://github.com/user-attachments/assets/d68e94cc-2dc6-4bc9-9a6c-4acb1fced89c)

After having a look at its shadow box used it seems there was empty space in kanban card that is why it is not getting clipped in there but is getting clipped calendar popover.

![2024-07-26_16-02](https://github.com/user-attachments/assets/58da030a-93a1-4c46-84e7-28d213a5390e)

That is why I have reduced the size of icon based on length and width provided within the css. What do you think?

Reply:
By doing that, you will also alter the `project_task_state_selection` component. 😕 

Also, I don't really understand why we did not have the issue before your changes. 🤔

Reply:
```project_task_state_selection``` is used in kanban card. Here the problem doesn't occur because there is empty space as shown in 2nd picture from above. But when it comes to calendar popover there is gap for the ```project_task_state_selection widget``` that is why it's getting clipped. I think its best if i write separate css for ```project_task_stage_with_state_selection```. What do you think?

Reply:
> `project_task_state_selection` is used in kanban card. Here the problem doesn't occur because there is empty space as shown in 2nd picture from above. But when it comes to calendar popover there is gap for the `project_task_state_selection widget` that is why it's getting clipped. I think its best if i write separate css for `project_task_stage_with_state_selection`. What do you think?

Before your changes `project_task_state_selection` was also displayed in the popover. 🤔

Reply:
I have found the same issue in master also :boom: 
![2024-07-31_09-51](https://github.com/user-attachments/assets/17e33f2f-4586-447c-ab9f-c276bde10716)

Reply:
Should I investigate more deep into this?